app/controllers/posts_controller.rb

```diff
@@ -16,9 +16,9 @@ def new
   def create
     @post = current_user.posts.build(post_params)
     if @post.save
-      redirect_to user_posts_path(current_user)
+      redirect_to user_posts_path(current_user), notice: 'You have successfully created a post'
     else
-      render :new
+      redirect_to new_post_path, notice: 'This is an invalid post'
     end
   end
 
```

Comment:
- [ ] You're expected to solve N+1 problem when fetching all posts and their comments for a user as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLAZ4K2ZHA777536NECDBHORQS) Example in the posts#index actions students should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`